src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/SdkLimitOptions.cs

```diff
@@ -31,7 +31,7 @@ internal sealed class SdkLimitOptions
     private int? spanLinkAttributeCountLimit;
     private bool spanLinkAttributeCountLimitSet;
 
-    internal SdkLimitOptions()
+    public SdkLimitOptions()
```

Comment:
Note: The class is still `internal` this just satisfies the `new()` constraint which Options 3.1 requires of TOptions.